charts/astronomer/templates/houston/worker/houston-worker-deployment.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         component: houston-worker
         release: {{ .Release.Name }}
       annotations:
-        checksum/houston-config: {{ include (print $.Template.BasePath "/houston/api/houston-configmap.yaml") . | sha256sum }}
+        checksum/houston-config: {{ include (print $.Template.BasePath "/houston/worker/houston-worker-configmap.yaml") . | sha256sum }}
```

Comment:
@adam2k you are using it here.
but i think you can use prod one and just disable commander via ENV_VARIABLE.

Reply:
I'm testing now without it to see. 

The other part of this question is does it make sense to reuse the `/houston/api/houston-configmap.yaml` chart for the houston worker?  Will it cause breaking changes later on if somebody changes `/houston/api/houston-configmap.yaml` and they don't test the changes in the worker?

Reply:
It seems like this file is required.  When I tried to revert this reference and delete the `/houston/worker/houston-worker-configmap.yaml` file I am getting a lot of crashLoopBackOff in the worker pod.

Reply:
I confirmed the steps in the main PR comment and it works as-is.